taker-frontend/src/components/Wallet.tsx

```diff
@@ -228,17 +279,119 @@ export default function Wallet(
                         >
                             Withdraw
                         </Button>
-                        <Button
-                            variant={"solid"}
-                            colorScheme={"blue"}
-                            onClick={exportSeed}
-                        >
-                            Backup Seed
-                        </Button>
                     </HStack>
                 </VStack>
 
-                <Divider marginTop={2} marginBottom={2} />
+                {environment !== "umbrel" && (
+                    <Box>
```

Comment:
Would be nice to get some screenshots in the PR description, my braincompiler is not showing me that nicely :)

Reply:
I've added a gif to the PR description.